### PR TITLE
fix: change session id for column lineage

### PR DESCRIPTION
**Pull request by AdiGajbhiye**
## Overview

- Using `env.sessionId` which will have values like `2fcc86de-6267-4471-8e04-07904223158b1716315453561`. It changes on vscode restart.
- Remove previous `sessionId`

### Problem

Describe the problem you are solving. Mention the ticket/issue if applicable.

### Solution

Describe the implemented solution. Add external references if needed.

### Screenshot/Demo

A picture is worth a thousand words. Please highlight the changes if applicable.

### How to test

- Steps to be followed to verify the solution or code changes
- Mention if there is any settings configuration added/changed/deleted

## Checklist

- [ ] I have run this code and it appears to resolve the stated issue
- [ ] `README.md` updated and added information about my change

### CODE REVIEW

**Review by mdesmet — APPROVED** [review]

File: src/webview_provider/newLineagePanel.ts [hunk 1]
```diff
@@ -639,14 +640,15 @@ export class NewLineagePanel implements LineagePanelView {
...
+      const session_id = `${env.sessionId}-${targets[0]}-${targets[1]}`;
       const request = {
         model_dialect: modelDialect,
         model_info: modelInfos,
         upstream_expansion: upstreamExpansion,
         targets: targets.map((t) => ({ uniqueId: t[0], column_name: t[1] })),
         selected_column: selected_column!,
         parent_models,
-        session_id: sessionId,
+        session_id,
```
Comment (suryaiyer95) [inline src/webview_provider/newLineagePanel.ts:651]: should it be session_id: sessionId?

Comment (suryaiyer95) [reply src/webview_provider/newLineagePanel.ts:651]: same for parent_models?

Comment (mdesmet) [reply src/webview_provider/newLineagePanel.ts:651]: Yeah, indeed should use camelcase as it is a simple JS variable, not a API object.

File: src/webview_provider/newLineagePanel.ts [hunk 2]
```diff
@@ -639,6 +640,7 @@ export class NewLineagePanel implements LineagePanelView {
       if (this.cancellationTokenSource?.token.isCancellationRequested) {
         return { column_lineage: [] };
       }
+      const sessionId = `${env.sessionId}-${selectedColumn.table}-${selectedColumn.name}`;
```
Comment (suryaiyer95) [inline src/webview_provider/newLineagePanel.ts:643]: do you have model unique_id available?

Comment (AdiGajbhiye) [reply src/webview_provider/newLineagePanel.ts:641]: `selectedColumn.table` should be that.

**Pull request merged**